docs/Lua.md

```diff
@@ -282,6 +446,15 @@ input.getMouseDelta() -- Returns vector
 ## Resource Manager
 The resources global that has been created in the lua environment is the main interface of the resource manager in C++. It is used to load in a variety of assets from disk, such as meshes and textures.
 
+With the resources global comes another global table that is related to Filter modes to textures. These value can be used in a table when [adding a texture](#addTexture) to the resource manager.
+~~~ Lua
+-- Filters
+Filters.Nearest
+Filters.Linear
+Filters.CubicEXT
```

Comment:
Cubic needs a vulkan extension enabled, so it will cause validation warnings if used right now. Might be good to know for the end user.